force-app/main/default/classes/GooglePlacesUtility.cls

```diff
@@ -4,46 +4,49 @@ public with sharing class GooglePlacesUtility {
     public static void validateInputs(Id account, Decimal longitude, Decimal latitude, Integer radius, String type) {
 
         if (!type.equalsIgnoreCase('school')) {
-            AuraHandledException e = new AuraHandledException('Type needs to be school.');
-            e.setMessage('Type needs to be school.');
-            throw e;
+            throwError('Type needs to be school.');
```

Comment:
@sungysang error label change: "Invalid Account Type. Type needs to be "Church"."